src/cdrizzlemap.c

```diff
@@ -619,12 +666,16 @@ clip_polygon_to_window(const struct polygon *p, const struct polygon *wnd,
                 // compute intersection point:
                 // https://en.wikipedia.org/wiki/Line–line_intersection
                 d = area(dp, dw);  // d != 0 because (v2_inside != v1_inside)
+
                 app_ = area(*pv, *pv_);
                 aww_ = area(*wv, *wv_);
+
                 vi.x = (app_ * dw.x - aww_ * dp.x) / d;
                 vi.y = (app_ * dw.y - aww_ * dp.y) / d;
 
-                append_vertex(ppout, vi);
+                if (isfinite(vi.x) && isfinite(vi.y)) {
```

Comment:
Is this the code change that fixed the bug (and all other changes are formatting or unused code)?

Reply:
yes